.github/workflows/sync_typeshed.yml

```diff
@@ -0,0 +1,31 @@
+name: Sync typeshed
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 1,15 * *"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync_typeshed:
+    name: Sync typeshed
+    if: github.repository == 'python/typeshed'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        # TODO: use whatever solution ends up working for
+        # https://github.com/python/typeshed/issues/8434
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: git config
+        run: |
+          git config --global user.name mypybot
+          git config --global user.email '<>'
+      - name: Sync typeshed
+        run: |
+          python -m pip install requests
```

Comment:
For security maybe we should pin all versions here

Reply:
Done, although if you're able to compromise a `requests` release you probably aren't going to use it to pwn the sync typeshed mypy workflow :-)

Reply:
Well, it's also all of the transitive dependencies. Though I suppose pinning all of those would be a pain.

Reply:
`requests` also infamously pins its own requirements pretty tightly, so there shouldn't be much of a security risk from transitive dependencies with `requests`

Reply:
People who sneak security bugs into packages are unlikely to respect semver.

Reply:
...True.